### PR TITLE
Port Enterprise Docs

### PR DESCRIPTION
**Pull request by acnagy**
it's happening! 🚀

### CODE REVIEW

File: user/enterprise/installation.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+statuses back to the platform. It must be installed on a separate machine
+instance from the Platform.
+
+### Creating a Security Group
+
+If you're setting up your AMI for the first time you will need to create
+a Security Group. From the EC2 management console, create an entry for
+each port in the table below:
+
+| Port | Service | Description |
```
Comment (danishkhan) [inline user/enterprise/installation.md]: enterprise installation - travis ci 2017-08-10 14-47-11

Can't figure out how to align that better. Not the end of the world if it stays that way, but was curious if anyone had some ideas

**Comment by plaindocs** [comment]
My main issue with merging this is that we won't have Enterprise-specific designs till next month.

@acnagy @danishkhan what do you think about waiting for that vs merging now? 

Either way I'll try to give it a quick once over this week.

**Comment by acnagy** [comment]
@plaindocs makes sense to me to merge for now - I think it will likely help more than not to merge these in sooner rather than later... we could certainly hold off as well

**Comment by danishkhan** [comment]
Edits look good to me 👍

File: user/enterprise.md [hunk 2]
```diff
@@ -19,7 +18,7 @@ to use LDAP or SAML to manage your users access. Travis CI Enterprise also
 integrates with GitHub.com, allowing you greater control over your security, and
 to scale out your build infrastructure as needed.
 
-### Why choose Enterprise over travis-ci.com?
+## Why choose Enterprise over travis-ci.com?
 
 Travis CI Enterprise is built to bring the features of the [hosted
```
Comment (MariadeAnton) [inline user/enterprise.md]: I'd add a clarification in this section or in the top, mentioning that Travis CI Enterprise is their in-house Travis CI solution? I'd hope that would help clarify any confusion until we get visually distinctive docs for hosted and enterprise :) cc/ @danishkhan is there a specific Enterprise wording we should use to do that? :)

File: user/enterprise/build-images.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+layout: en_enterprise
+
+---
+
+With Travis CI Enterprise, you can configure your build images to suit your
+development process and improve the build environment and performance.
+
+<div id="toc"></div>
+
+## Customizing build images
```
Comment (MariadeAnton) [inline user/enterprise/build-images.md]: 🎉

**Comment by acnagy** [comment]
@plaindocs ✋ I commit to fixing the TODOs - you can even see the block on my calendar, if you want

Agreed on the redirect too - I'm not sure if I'm gonna slash and burn and just do a big Docs --> Docs redirect, but again, can figure that out in a bit. 

So...Shall we 🚀?

**Pull request merged**